cpu/cortexm_common/Makefile.include

```diff
@@ -1,2 +1,4 @@
 # include module specific includes
 export INCLUDES += -I$(RIOTCPU)/cortexm_common/include
+
+USEMODULE += cortexm_common_periph
```

Comment:
Nope. The decision if the layered module should be used is up to a CPU. Some cortexm CPUs might want to use it (e.g. stms), but others might opt out (e.g. nrf5x)...